src/components/Markdown/Headings.tsx

```diff
@@ -49,6 +49,7 @@ export const Headings: FunctionComponent<HeadingResolverProps> = ({
         data-heading-title={title}
         id={id}
         replace
+        sx={{ "> code": { color: "black", fontSize: "inherit" } }}
```

Comment:
IMO we should keep the color within our theme & consistent with other headings which is `blue.800`

Reply:
Fixed 👍